documentation/enterprise/enterprise-distribution.md

```diff
@@ -0,0 +1,312 @@
+
+The alternative, and recommended, approach is to use the option for unlisted add-ons on [addons.mozilla.org](https://addons.mozilla.org) (AMO). This option means that you can get a signed add-on without it being listed in the public add-ons directory. This feature provides a signed add-on immediately. This signed add-on can then be installed from a web page behind the firewall, or installed using one of the options described here.
+
+{% endcapture %}
+{% include modules/column-w-toc.html
+  id="signed-vs-unsigned"
+  content=content_with_toc
+%}
+
+<!-- END: Content with Table of Contents -->
+
+<!-- Single Column Body Module -->
+
+{% capture content %}
+
+## Sideloading
+
+You can sideload an add-on using one of the standard extensions folders, as described in [Installation using standard extension folders](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/Distribution_options/Sideloading_add-ons#Installation_using_the_standard_extension_folders).
+
+{% endcapture %}
+{% include modules/one-column.html
+  id="sideloading"
+  content=content
+  aside=""
+%}
+
+<!-- END: Single Column Body Module -->
+
+<!-- Single Column Body Module -->
+
+{% capture content %}
+
+## Installation using the Windows registry
+
+This section explains how to install add-ons into Firefox using the Windows Registry.
+
+<!-- Alert -->
+
+{% capture alert %}
+
+Before Firefox 62 it was possible to load unpacked extensions by making the Windows registry key point to a directory containing an unpackaged extension.
+
+From Firefox 62 this is no longer possible, and the key must point to a packaged XPI file, as described in this section.
+
+{% endcapture %}
+{% include modules/note.html
+	content=alert
+	alert=true
+%}
+
+<!-- END: Alert -->
+
+<!-- Note -->
+
+{% capture note %}
+
+It is safe to modify the Registry keys while Firefox is running.
+
+{% endcapture %}
+{% include modules/note.html
+	content=note
+	alert=false
+%}
+
+<!-- END: Note -->
+
+1. Ensure the add-on has an add-on ID, by including the following to its manifest.json file, replacing _your-add-on-name@your-domain.com_ with a suitable ID for your add-on:
+
+<!-- Syntax Highlighting -->
+
+{% highlight javascript linenos %}
+"applications": {
+"gecko": {
+"id": "your-add-on-name@your-domain.com"
+}
+}
+{% endhighlight %}
+
+<!-- END: Syntax Highlighting -->
+
+An email address style ID is recommended.
+
+2. Sign your add-on on addons.mozilla.org (AMO) using the unlisted option. For more details, see Signing and distributing your add-on.
+
+3. Download the signed XPI file and ensure the file name is the add-on ID plus the extension .xpi. For example, `c:/webext/borderify@example.com.xpi`
+
+4. Open Regedit and add keys as follows:
+
+   - For all users of the computer, add to the following registry keys:
+     `HKEY_LOCAL_MACHINE\Software\Mozilla\Firefox\Extensions`
```

Comment:
looks like the line break didn't work (add one more line?) 
```suggestion
    
    `HKEY_LOCAL_MACHINE\Software\Mozilla\Firefox\Extensions`
```
![Screenshot_2019-08-15 Enterprise distribution(1)](https://user-images.githubusercontent.com/28877053/63110971-78ae9e80-bf41-11e9-8b12-20b9fc793067.png)